<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>660</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(56, 56, 56);</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="link_text">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>701</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <pointsize>14</pointsize>
      <weight>7</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit{
	background-color: rgb(255, 255, 255);
	font: 63 14pt &quot;Segoe UI Semibold&quot;;
	border-radius: 20px;
}

QTextEdit:hover {
	background-color: rgb(202, 202, 202);
	color: rgb(56, 56, 56);
}
</string>
    </property>
    <property name="locale">
     <locale language="Duala" country="Cameroon"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
    <property name="midLineWidth">
     <number>5</number>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="autoFormatting">
     <set>QTextEdit::AutoBulletList</set>
    </property>
    <property name="tabChangesFocus">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI Semibold'; font-size:14pt; font-weight:56; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="tabStopWidth">
     <number>80</number>
    </property>
    <property name="placeholderText">
     <string>                                               לינק לפוסט</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start_btn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>250</y>
      <width>511</width>
      <height>271</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	font: 63 30pt &quot;Segoe UI Semibold&quot;;
	border-radius: 60px;
}

QPushButton:hover {
	background-color: rgb(255, 255, 255);
	color: rgb(56, 56, 56);
}

QPushButton:pressed {
	
	background-color: rgb(85, 255, 127);
	color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>חלץ תגובות</string>
    </property>
   </widget>
   <widget class="QLabel" name="headline_brit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/britney/britney.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="information_lbl">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>600</y>
      <width>721</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 14pt &quot;Segoe UI Semibold&quot;;
color: rgb(170, 255, 255);
color: rgb(255, 255, 127);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#ffff7f;&quot;&gt;קובץ האקסל יישמר באותה התיקייה שבה נמצאת האפליקציה&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
   <widget class="QPushButton" name="restart_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	image: url(:/britney/restart.png);
	border: 0px solid;
	border-radius: 25px;
}


QPushButton:hover {
	background-color: rgb(255, 255, 255);
}

QPushButton:pressed {
	
	background-color: rgb(85, 255, 127);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="headline_lbl">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 14pt &quot;Segoe UI Semibold&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Leave britney Alone..!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="mid_brit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>421</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/britney/britney.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="loader_lbl">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>571</width>
      <height>481</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>start_btn</zorder>
   <zorder>mid_brit</zorder>
   <zorder>loader_lbl</zorder>
   <zorder>link_text</zorder>
   <zorder>headline_brit</zorder>
   <zorder>information_lbl</zorder>
   <zorder>restart_btn</zorder>
   <zorder>headline_lbl</zorder>
  </widget>
 </widget>
 <resources>
  <include location="images/Britney.qrc"/>
 </resources>
 <connections/>
</ui>
